### DISCUSSION TITLE
Too transparent and hard to see.

### DISCUSSION THREAD

**Discussion by Bill-Costa** [will-stone/browserosaurus#511] [Q&A]
Running macOS High Sierra (10.13) -- yes I know, very old.  Perhaps this is why the pop ups are super transparent and can barely be seen.  Any chance turning off transparency as an option?

**Comment by will-stone** [comment]
Hi. Can you please provide a screenshot? Also, what happens if you select “reduce transparency” in your system’s accessibility settings? Does that make it better?
